velox/functions/lib/window/tests/WindowTestBase.h

```diff
@@ -19,6 +19,8 @@
 
 namespace facebook::velox::window::test {
 
+enum class WindowStyle { SORT, STREAMING, RANDOM };
```

Comment:
Enum values start with k and are camelCase so kSort, kStreaming and kRandom

Reply:
Updated.